tests/src/commands/deposit.spec.ts

```diff
@@ -0,0 +1,119 @@
+import { BigNumber } from "ethers";
+import { parseEther } from "ethers/lib/utils";
+
+import { handler as deposit } from "../../../src/commands/deposit";
+import { mockConsoleOutput } from "../../utils/mockers";
+import { mockL2Provider, mockL1Provider, mockL2Wallet } from "../../utils/mocks";
+
+describe("deposit", () => {
+  let stdOutMock: ReturnType<typeof mockConsoleOutput>;
+  let l1ProviderMock: ReturnType<typeof mockL1Provider>;
+  let l2ProviderMock: ReturnType<typeof mockL2Provider>;
+  let l2WalletMock: ReturnType<typeof mockL2Wallet>;
+
+  const chain = "era-testnet";
+  const amount = "0.1";
+  const privateKey = "7726827caac94a7f9e1b160f7ea819f172f7b6f9d2a97f992c38edeab82d4110";
+  const recipient = "0xa1cf087DB965Ab02Fb3CFaCe1f5c63935815f044";
+  const senderFinalBalance = "0.5";
+  const transactionHash = "0x5313817e1e3ba46e12aad81d481293069096ade97b577d175c34a18466f97e5a";
+
+  beforeEach(() => {
+    stdOutMock = mockConsoleOutput();
+    l1ProviderMock = mockL1Provider({
+      getBalance: jest.fn().mockResolvedValue(BigNumber.from(parseEther(senderFinalBalance))),
+    });
+    l2ProviderMock = mockL2Provider();
+    l2WalletMock = mockL2Wallet({
+      deposit: () => Promise.resolve({ hash: transactionHash }),
+    });
+  });
+
+  afterEach(() => {
+    stdOutMock.mockRestore();
+    l1ProviderMock.mockRestore();
+    l2ProviderMock.mockRestore();
+    l2WalletMock.mockRestore();
+  });
+
+  it("runs deposit method with defined parameters", async () => {
+    const depositMock = jest.fn().mockResolvedValue({
+      hash: transactionHash,
+    });
+    l2WalletMock = mockL2Wallet({
+      deposit: depositMock,
+    });
+
+    await deposit({
+      amount: amount,
+      chain,
+      recipient,
+      privateKey,
+    });
+
+    expect(depositMock).toHaveBeenCalledWith({
+      amount: BigNumber.from(parseEther(amount)),
+      to: recipient,
+      token: "0x0000000000000000000000000000000000000000",
+    });
+  });
+  it("outputs expected logs", async () => {
+    await deposit({
+      amount: amount,
+      chain,
+      recipient,
+      privateKey,
+    });
+
+    expect(stdOutMock).not.hasConsoleErrors();
+
+    expect(stdOutMock).toBeInConsole("Deposit:");
+    expect(stdOutMock).toBeInConsole("From: 0x36615Cf349d7F6344891B1e7CA7C72883F5dc049 (Ethereum Goerli)");
```

Comment:
We should move the public address to a const at the top with a name that better shows where this value comes from:
```suggestion
    expect(stdOutMock).toBeInConsole(`From: ${publicAddress} (Ethereum Goerli)`);
```

NOTE: This happens in a lot of the tests, but just adding one comment to not spam the same thing in this PR review